src/wkt/tests/message_with_string.rs

```diff
@@ -229,21 +229,29 @@ mod tests {
         Ok(())
     }
 
-    #[test_case(json!({}))]
-    #[test_case(json!({"unknown0": "hello", "unknown1": "world"}))]
-    fn test_empty_unknown_fields(input: Value) -> Result {
-        let got = serde_json::from_value::<MessageWithString>(input.clone())?;
-        let unknown_fields = serde_json::from_value::<
-            serde_json::Map<std::string::String, serde_json::Value>,
-        >(input)?;
-        if unknown_fields.is_empty() {
-            assert!(!format!("{got:?}").contains("_unknown_fields"));
-            return Ok(());
-        }
-        for (key, value) in unknown_fields.into_iter() {
-            let value_str = serde_json::from_value::<String>(value)?;
-            assert!(format!("{got:?}").contains(&key));
-            assert!(format!("{got:?}").contains(&value_str));
+    #[test]
+    fn test_debug_without_unknown_fields() -> Result {
+        let got = MessageWithString::new()
+            .set_singular("hello")
+            .set_optional("world");
+        let debug_str = format!("{got:?}");
+        assert!(!debug_str.contains("_unknown_fields"));
+        assert!(debug_str.contains("singular: \"hello\""));
+        assert!(debug_str.contains("optional: Some(\"world\")"));
```

Comment:
nit: consider using a raw string literal when dealing with escaped characters like quotes.

```suggestion
        assert!(debug_str.contains(r#"singular: "hello""#));
        assert!(debug_str.contains(r#"optional: Some("world")"#));
```

https://doc.rust-lang.org/rust-by-example/std/str.html#literals-and-escapes